<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Busca</class>
 <widget class="QDialog" name="Busca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>564</width>
    <height>543</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Buscador</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recoverdrake.qrc">
    <normaloff>:/Imagenes/lupa.png</normaloff>:/Imagenes/lupa.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Opciones de busqueda</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Mayusculas/Minusculas</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="toolTip">
            <string>No distingir entre mayusculas y minusculas</string>
           </property>
           <property name="statusTip">
            <string>No distingir entre mayusculas y minusculas</string>
           </property>
           <property name="whatsThis">
            <string>No distingir entre mayusculas y minusculas</string>
           </property>
           <property name="text">
            <string>No distingir Mayusculas/Minusculas</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="toolTip">
            <string>Distingir entre mayusculas y minusculas</string>
           </property>
           <property name="statusTip">
            <string>Distingir entre mayusculas y minusculas</string>
           </property>
           <property name="whatsThis">
            <string>Distingir entre mayusculas y minusculas</string>
           </property>
           <property name="text">
            <string>Distingir Mayusculas/Minusculas</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Desde donde buscar</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="toolTip">
            <string>Ruta desde donde buscar</string>
           </property>
           <property name="statusTip">
            <string>Ruta desde donde buscar</string>
           </property>
           <property name="whatsThis">
            <string>Ruta desde donde buscar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Acceso a directorio</string>
           </property>
           <property name="statusTip">
            <string>Acceso a directorio</string>
           </property>
           <property name="whatsThis">
            <string>Acceso a directorio</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Peso/Tipo</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_4">
           <property name="toolTip">
            <string>Buscar condicionales por peso de ficheros</string>
           </property>
           <property name="statusTip">
            <string>Buscar condicionales por peso de ficheros</string>
           </property>
           <property name="whatsThis">
            <string>Buscar condicionales por peso de ficheros</string>
           </property>
           <property name="text">
            <string>Buscar por peso de ficheros</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Peso</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="3">
             <widget class="QRadioButton" name="radioButton_4">
              <property name="toolTip">
               <string>Menor que...</string>
              </property>
              <property name="statusTip">
               <string>Menor que...</string>
              </property>
              <property name="whatsThis">
               <string>Menor que...</string>
              </property>
              <property name="text">
               <string>Menor</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="radioButton_3">
              <property name="toolTip">
               <string>Mayor que...</string>
              </property>
              <property name="statusTip">
               <string>Mayor que...</string>
              </property>
              <property name="whatsThis">
               <string>Mayor que...</string>
              </property>
              <property name="text">
               <string>Mayor</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Cantidad a buscar</string>
              </property>
              <property name="statusTip">
               <string>Cantidad a buscar</string>
              </property>
              <property name="whatsThis">
               <string>Cantidad a buscar</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QRadioButton" name="radioButton_5">
              <property name="toolTip">
               <string>Igual que...</string>
              </property>
              <property name="statusTip">
               <string>Igual que...</string>
              </property>
              <property name="whatsThis">
               <string>Igual que...</string>
              </property>
              <property name="text">
               <string>Igual</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="4">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="toolTip">
            <string>Tipo de busqueda</string>
           </property>
           <property name="statusTip">
            <string>Tipo de busqueda</string>
           </property>
           <property name="whatsThis">
            <string>Tipo de busqueda</string>
           </property>
           <property name="title">
            <string>Tipo</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_6">
              <property name="toolTip">
               <string>Busqueda por Kb</string>
              </property>
              <property name="statusTip">
               <string>Busqueda por Kb</string>
              </property>
              <property name="whatsThis">
               <string>Busqueda por Kb</string>
              </property>
              <property name="text">
               <string>Kilobyte</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="radioButton_7">
              <property name="toolTip">
               <string>Busqueda por Mb</string>
              </property>
              <property name="statusTip">
               <string>Busqueda por Mb</string>
              </property>
              <property name="whatsThis">
               <string>Busqueda por Mb</string>
              </property>
              <property name="text">
               <string>Megas</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="radioButton_8">
              <property name="toolTip">
               <string>Busqueda por Gb</string>
              </property>
              <property name="statusTip">
               <string>Busqueda por Gb</string>
              </property>
              <property name="whatsThis">
               <string>Busqueda por Gb</string>
              </property>
              <property name="text">
               <string>Gigas</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Sans Serif&quot;;
font: bold;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Abrir ruta</string>
     </property>
     <property name="icon">
      <iconset resource="recoverdrake.qrc">
       <normaloff>:/Imagenes/lupaO.png</normaloff>:/Imagenes/lupaO.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_3">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Nos vamos...</string>
     </property>
     <property name="statusTip">
      <string>Nos vamos...</string>
     </property>
     <property name="whatsThis">
      <string>Nos vamos...</string>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="icon">
      <iconset resource="recoverdrake.qrc">
       <normaloff>:/Imagenes/puerta.png</normaloff>:/Imagenes/puerta.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>Tipo de busqueda</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_9">
        <property name="text">
         <string>Por nombre</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_10">
          <property name="text">
           <string>Por espacio (los 10 archivos/carpetas que mas ocupan)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Buscar</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/lupaO.png</normaloff>:/Imagenes/lupaO.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Nombre a buscar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="toolTip">
         <string>Palabra clave a buscar</string>
        </property>
        <property name="statusTip">
         <string>Palabra clave a buscar</string>
        </property>
        <property name="whatsThis">
         <string>Palabra clave a buscar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 255);
font: bold;</string>
        </property>
        <property name="text">
         <string>*Utiliza comodines para ampliar la busqueda. (Ejem. *mos*.avi, *mos*, *.avi, *mos o mos*)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Lanzar la busqueda</string>
        </property>
        <property name="statusTip">
         <string>Lanzar la busqueda</string>
        </property>
        <property name="whatsThis">
         <string>Lanzar la busqueda</string>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/lupa.png</normaloff>:/Imagenes/lupa.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_5">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Cancelar la busqueda comenzada</string>
        </property>
        <property name="statusTip">
         <string>Cancelar la busqueda comenzada</string>
        </property>
        <property name="whatsThis">
         <string>Cancelar la busqueda comenzada</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ejecutar los archivos localizados</string>
        </property>
        <property name="statusTip">
         <string>Ejecutar los archivos localizados</string>
        </property>
        <property name="whatsThis">
         <string>Ejecutar los archivos localizados</string>
        </property>
        <property name="text">
         <string>Ejecutar</string>
        </property>
        <property name="icon">
         <iconset resource="recoverdrake.qrc">
          <normaloff>:/Imagenes/tirachinas.png</normaloff>:/Imagenes/tirachinas.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="toolTip">
      <string>Fichero localizados en la busqueda</string>
     </property>
     <property name="statusTip">
      <string>Fichero localizados en la busqueda</string>
     </property>
     <property name="whatsThis">
      <string>Fichero localizados en la busqueda</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Ruta/Archivo</string>
      </property>
      <property name="icon">
       <iconset resource="recoverdrake.qrc">
        <normaloff>:/Imagenes/64px-Dolphin-icon.svg.png</normaloff>:/Imagenes/64px-Dolphin-icon.svg.png</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Peso</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recoverdrake.qrc"/>
 </resources>
 <connections/>
</ui>
